tests/integration/components/sub-nav/component-test.js

```diff
@@ -0,0 +1,26 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Component | sub-nav', function(hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function(assert) {
```

Comment:
feel free to add tests and try it out

Reply:
Thank you Shaun! I added a couple tests.